dashboard/test/controllers/openai_sessions_controller_test.rb

```diff
@@ -0,0 +1,32 @@
+require 'test_helper'
+
+class OpenaiSessionsControllerTest < ActionController::TestCase
+  include OpenaiChatHelper
```

Comment:
For my understanding, is this necessary?

Reply:
It's not! Thanks - will remove.

Reply:
Looks like this didn't get removed?